GitUI/CommandsDialogs/FormVerify.cs

```diff
@@ -216,6 +218,68 @@ private void Warnings_CellMouseDoubleClick(object sender, DataGridViewCellMouseE
             ViewCurrentItem();
         }
 
+        private void Warnings_SelectionChanged(object sender, EventArgs e)
+        {
+            if (CurrentItem == null || _previewedItem == CurrentItem)
+            {
+                return;
+            }
+
+            _previewedItem = CurrentItem;
+
+            var content = Module.ShowObject(_previewedItem.ObjectId);
+            if (_previewedItem.ObjectType == LostObjectType.Commit)
+            {
+                ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+                {
+                    await fileViewer.ViewPatchAsync("commit.patch", content, null);
+                }).FileAndForget();
```

Comment:
I think it could be this
```suggestion
                ThreadHelper.JoinableTaskFactory.RunAsync(() => fileViewer.ViewPatchAsync("commit.patch", content, null))
                    .FileAndForget();
```